de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,111 @@
+app-id: de.nagstamon.nagstamon
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: nagstamon
+rename-desktop-file: nagstamon.desktop
+rename-icon: nagstamon
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=.nagstamon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
```

Comment:
Are these two needed? They should go through the portal if the app/toolkit supports it.

Reply:
According to https://flatpak-docs.readthedocs.io/en/latest/portals-qt.html the portal is not supported.